gen3/neural-networks/advanced-examples/ocr/ocr/README.md

```diff
@@ -1,31 +1,39 @@
-## Text Detection + Optical Character Recognition (OCR) Pipeline
+# Overview
+We provide here an example for running a two stage text detection and OCR pipeline. This example uses PaddlePaddle text detection and text recognition (OCR) models from [HubAI Model ZOO](https://hub.luxonis.com/ai/models). The example visualizes the recognized text on an adjacent white image in the locations in which it was detected. This example showcases how a twostage pipeline can easily be built using depthai.
```

Comment:
Instead of linking to the ZOO lets rather link to both specific models used in this examle